### ISSUE TITLE
Consider renaming simulate to semantics

### ISSUE THREAD

**Issue by satnam6502** [project-oak/silveroak#807]
Upon reflection, the word `simulate` is perhaps not the best choice and maybe we should revert to `semantics` or some other name that causes less confusion.

**Comment by jadephilipoom** [comment]
I agree that the name is a bit confusing, but `semantics` is currently what we call our Cava instances when defining circuits, so that particular replacement might cause name collisions. I'm struggling to think of other ideas -- `run` and `interp` both seem like they would cause just as much confusion.

**Comment by satnam6502** [comment]
Good point. Let's noodle on this a bit: it's not urgent.
